ui/src/shared/middleware/resizeLayout.js

```diff
@@ -12,5 +13,9 @@ export default function resizeLayout() {
       evt.initEvent('resize', false, true)
       window.dispatchEvent(evt)
     }
+    if (window.location.search.includes('present')) {
+      next(enablePresentationMode())
```

Comment:
Straightforward, nice. Shows up as an action in Redux devtools, which is a plus.

Reply:
yep! 👍